### DISCUSSION TITLE
How can I change the Content Background Color of a TabPanel (TabView)?

### DISCUSSION THREAD

**Discussion by tomsoja978** [primefaces/community#867] [PrimeReact] [answered]
How can I change the Content Background Color of a TabPanel (TabView) in Primereact (JS) ? Ideally to the basic background color of the page, which is black here (I guess). I think I have to modify css, or, maybe add some attribute in a style="..." way, but I am not familiar at all when it comes to styling (Sorry).

Appreciate any suggestions. Thank you.

https://primereact.org/tabview/

![image](https://github.com/primefaces/community/assets/144290567/16ad13ee-4f35-4473-943d-1f0e07fe1011)

**Comment by melloware** [comment, marked as answer]
Just add this CSS to whatever color you want:

```css
.p-tabview-panel {
  background-color: green;
}
```

**Reply by tomsoja978** [reply]
in particular, i had to set it to 'none'

Thanks!

**Reply by melloware** [reply]
Dont forget to mark my Stack Overflow answer as the Answer for others to see: https://stackoverflow.com/questions/77786296/how-can-i-change-the-content-background-color-of-a-tabpanel-tabview-in-primere
